pkgs/os-specific/linux/bpftrace/default.nix

```diff
@@ -4,12 +4,20 @@
 , libelf, libbfd, libbpf, libopcodes, bcc
 , cereal, asciidoctor
 , nixosTests
+, util-linux
 }:
 
 stdenv.mkDerivation rec {
   pname = "bpftrace";
   version = "0.14.1";
 
+  # Rename *.8 to *.bt.8 to avoid conflicts with original bcc programs
+  # Cherry-picked from merged PR, remove this hook on next update
+  # https://github.com/iovisor/bpftrace/pull/2242
+  postUnpack = ''
+    ${util-linux}/bin/rename .8 .bt.8 "$sourceRoot"/man/man8/*.8
+  '';
```

Comment:
We cannot fetchpatch the linked commit?

Reply:
This is shorter, simpler and clearer.

Reply:
Then please fetch the patch. Using a rename utlity to emulate the patch is not simpler.

Reply:
The issue I see with not using `fetchpatch` is that on the next release update, someone might just merge the new version generated by ryantm without removing the fixup. And then manpages have are renamed to `.bt.bt.8`. With the patch in place the new build would fail if you don't remove it.

Reply:
I've checked the patch applies, so yeah I guess it is better (the last release was quite old so was worried there'd be new man pages since but it doesn't look like there are)

Also note 0.15.0 got released a few days ago, but unfortunately it doesn't include this patch -- so the next update will not be removing this, but the one after it will.

(I'll update bpftrace to 0.15.0 this weekend unless someone beats me to it, from a quick look it should work without problem after removing `btf-dump-new-0.6.0.patch` that is no longer needed)

Reply:
Alright, I'll switch to `fetchpatch`, thanks.

Reply:
`fetchpatch` can't cope with GitHub's summarised patch files containing "renamed this to that without content changes" rather than a full unified diff:

```
trying https://github.com/iovisor/bpftrace/commit/2df807dbae4037aa8bf0afc03f52fb3f6321c62a.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9671  100  9671    0     0   5433      0  0:00:01  0:00:01 --:--:--  5433
error: Normalized patch '/build/patch' is empty (while the fetched file was not)!
Did you maybe fetch a HTML representation of a patch instead of a raw patch?

[ patch content omitted, see link above ]

note: keeping build directory '/tmp/nix-build-rename-manuals-to-avoid-conflicts-with-bcc.patch.drv-0'
error: builder for '/nix/store/ik47f913bwqcyxr6mknd9y1sdmx5c1mh-rename-manuals-to-avoid-conflicts-with-bcc.patch.drv' failed with exit code 1;
```

Reply:
I guess than we have to live with the current variant.